include/z64animation.h

```diff
@@ -5,57 +5,120 @@
 #include "z64dma.h"
 #include "z64math.h"
 
-#define LINK_ANIMETION_OFFSET(addr,offset) (((u32)&_link_animetionSegmentRomStart)+((u32)addr)-((u32)&link_animetion_segment)+((u32)offset))
+#define LINK_ANIMATION_OFFSET(addr,offset) (((u32)&_link_animetionSegmentRomStart)+((u32)addr)-((u32)&link_animetion_segment)+((u32)offset))
 #define LIMB_DONE 0xFF
 #define ANIMATION_ENTRY_MAX 50
 
 #define ANIM_FLAG_UPDATEXZ 0x02
 #define ANIM_FLAG_UPDATEY  0x10
 
+extern u32 link_animetion_segment;
 struct GlobalContext;
 struct Actor;
 typedef struct SkelAnime SkelAnime;
 
 typedef struct {
-    /* 0x000 */ Vec3s translation;      // Translation relative to parent limb.  root limb is a tranlation for entire model.
-    /* 0x006 */ u8 firstChildIndex;     // The first child's index into the limb table.
-    /* 0x007 */ u8 nextLimbIndex;       // The parent limb's next limb index into the limb table.
-    /* 0x008 */ Gfx* displayLists[1];   // Display lists for the limb. Index 0 is the normal display list, index 1 is the far model display list.
-} SkelLimbEntry; // Size = 0xC or 0x10
+    /* 0x00 */ Vec3s translation;      // Translation relative to parent limb.  root limb is a tranlation for entire model.
+    /* 0x06 */ u8 firstChildIndex;     // The first child's index into the limb table.
+    /* 0x07 */ u8 nextLimbIndex;       // The parent limb's next limb index into the limb table.    
+} SkelLimb; // Size = 0x8
```

Comment:
Petrie and I discussed this in discord but want to put it here and get other opinions:
We were discussing the possibility of calling this `Bone` and switching the terminology from limb to bone in general.

This would mean that when something takes a "skeleton" its really just a pointer to the root bone which connects to everything else.
The term bone is apparently the more common terminology for this in video games, and I think it sounds good for the most part. I do think that saying for example "drawing a limb" makes more sense than "drawing a bone", since the display list for a given limb/bone draws the whole limb of an actor, and thinking about that aspect as a bone sounds a bit weird.
But, I really do like the continuity of the whole thing being called the skeleton and the parts being called a bone

relevant discussion is here: https://discordapp.com/channels/688807550715560050/688851317593997489/780217593407471636